graphite-clickhouse.go

```diff
@@ -117,8 +119,41 @@ func main() {
 	printVersion := flag.Bool("version", false, "Print version")
 	verbose := flag.Bool("verbose", false, "Verbose (print config on startup)")
 
+	match := flag.String(
+		"match",
+		"",
+		"Match metric rollup rules (for rollup config file)",
+	)
+
 	flag.Parse()
 
+	if *match != "" {
+		metric, ageStr, _ := strings.Cut(*match, " ")
+		age, _ := strconv.ParseUint(ageStr, 10, 32)
+		// check metric roolup rules
+		if rollup, err := rollup.NewXMLFile(*configFile, 0, ""); err == nil {
```

Comment:
I think it is supposed to parse rollup config instead of graphite-clickhouse config. Flag "-match" doesn't work locally for me.

Reply:
May be better way - use dirrerent command instead of default. Golang flag can't do this, but other solutions (pflag, cobra) can.